src/main/resources/gitlab/plugin.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<go-plugin id="gitlab.pr" version="1">
+    <about>
+        <name>Gitlab Pull Requests Builder</name>
+        <version>${version}</version>
+        <target-go-version>15.1.0</target-go-version>
```

Comment:
You might want to update this to a more latest version since you're using more latest JDK dependency :)

Reply:
Sure will update the version.